tests/unit/domain/test_metadata.py

```diff
@@ -171,31 +178,41 @@ def test_find_all_datastore_versions_when_draft_version_empty(mocker):
     assert len(actual['versions']) == 1
     assert actual['versions'][0]['version'] == '1.0.0.0'
 
-def test_find_all_metadata_skip_code_list_and_missing_values_for_data_structures(mocker):
+def test_find_all_metadata_skip_code_list_and_missing_values_for_metadata_all(mocker):
     
-    with open(DATA_STRUCTURES_FILE_PATH, encoding='utf-8') as f:
-        mocked_data_structures = json.load(f)
+    with open(METADATA_ALL_FILE_PATH, encoding='utf-8') as f:
+        mocked_metadata_all = json.load(f)
 
     mocker.patch.object(
         datastore, 'get_metadata_all',
-        return_value=mocked_data_structures
+        return_value=mocked_metadata_all
     )
     
     filtered_metadata = metadata.find_all_metadata_skip_code_list_and_missing_values(version='1.0.0.0')
-    __assert_code_list_and_missing_values(filtered_metadata)
+    __assert_code_list_and_missing_values(filtered_metadata['dataStructures'])
+
+    with open(METADATA_ALL_NO_CODE_LIST_FILE_PATH, encoding='utf-8') as f:
+        metadata_no_code_list = json.load(f)
+
+    assert metadata_no_code_list == filtered_metadata
 
 def test_find_all_metadata_skip_code_list_and_missing_values_for_data_structures(mocker):
     
-    with open(METADATA_ALL_FILE_PATH, encoding='utf-8') as f:
-        mocked_metadata_all = json.load(f)
+    with open(DATA_STRUCTURES_FILE_PATH, encoding='utf-8') as f:
+        mocked_data_structures = json.load(f)
 
     mocker.patch.object(
         datastore, 'get_metadata_all',
-        return_value=mocked_metadata_all
+        return_value=mocked_data_structures
     )
     
     filtered_metadata = metadata.find_all_metadata_skip_code_list_and_missing_values(version='1.0.0.0')
```

Comment:
Am I reading this wrong or is this now only returning the metadata_all['dataStructures']-list? Is it not supposed to return the full metadata object? 🤔

Reply:
Could you please run the tests locally, it seems fine to me.
What may be confusing is that metadata_all wraps all the other stuff under 'dataStructures'  while this level does not exist in data structures. I've also tested with postman with the expected outcome. (will double check ASAP though)